Managers/UTMQemuVirtualMachine.m

```diff
@@ -299,6 +299,15 @@ - (void)_vmStartWithCompletion:(void (^)(NSError * _Nullable))completion {
         }
     }
     
+    if (self.isRunningAsSnapshot) {
+        self.system.runAsSnapshot = self.isRunningAsSnapshot;
+    } else {
+        // Loading save states isn't possible when -snapshot is used
```

Comment:
> Loading save states isn't possible when -snapshot is used

Should this be exposed to the user somehow?

Reply:
This was discussed in Discord, specifying "Run without saving changes" is an explicit user choice so it overrides the loading of a save state.

Do you mean there should be a warning that the save state will be lost?

Reply:
Yes, that’s what I was going for.

Reply:
Anddddd it turns out my brain isn't running correctly...

This is basically just a safeguard, but the context menu UI does not show the "Run" or "Run without saving changes" button when there is a save state.

idk how I completely forgot about it and spent 15 mins implementing an alert... :(